### ISSUE TITLE
The co-teaching loss

### ISSUE THREAD

**Issue by badbadcode** [a5507203/Instance-dependent-Label-noise-Learning-under-a-Structural-Causal-Model#6]
Thanks for your brilliant work!

I want to know why the co-teaching loss is computed by the cross-entropy<c_logits, noisy labels> in your code.
https://github.com/a5507203/Instance-dependent-Label-noise-Learning-under-a-Structural-Causal-Model/blob/0129403e47cba3789756e3cfda849fc5cd5c704a/causalNL.py#L122
https://github.com/a5507203/Instance-dependent-Label-noise-Learning-under-a-Structural-Causal-Model/blob/0129403e47cba3789756e3cfda849fc5cd5c704a/causalNL.py#L57

Since it distinguishes the true labels and noisy labels during model training, why do we still use noisy labels to supervise the prediction of true labels（y_encoder）? 

If I have misunderstood something, please let me know.
